distribution/docs/src/main/resources/content/_running/starting-intro.adoc

```diff
@@ -201,11 +201,10 @@ root${at-symbol}localhost# chkconfig ${branding-lowercase}-service --add
 root${at-symbol}localhost# chkconfig ${branding-lowercase}-service on
 ----
 +
-*Redhat 7 (systemd)*
+*Redhat 7 or Centos 7 (systemd)*
 +
 ----
-root${at-symbol}localhost# systemctl enable ${home_directory}/bin/${branding-lowercase}-service
-----
+root${at-symbol}localhost# systemctl enable <${branding}_HOME>/bin/${branding-lowercase}.service
```

Comment:
same comment about home_directory

Reply:
I think this might have been an unintentional change from rebasing this changeset.